src/pages/InsureePage.js

```diff
@@ -39,7 +39,8 @@ class InsureePage extends Component {
 
   render() {
     const { classes, modulesManager, history, rights, insuree_uuid, family_uuid } = this.props;
-    if (!rights.includes(RIGHT_INSUREE_EDIT)) return null;
+    if (!rights.includes(RIGHT_INSUREE)) return null;
+    console.log(rights.includes(RIGHT_INSUREE_EDIT), !rights.includes(RIGHT_INSUREE_ADD))
```

Comment:
Do we still need the debug infos?

Reply:
It's not not necessary, I removed it from code.